### ISSUE TITLE
CIFAR100 Results 

### ISSUE THREAD

**Issue by NicoNico6** [szq0214/S2-BNN#3]
Hi, I noticed that in Table 5, the accuracy of training from scratch is only 70.9% for full precision ReActNet-MobileNet and 37.2% for binary. May I ask why the accuracy for the binary version is so low? According to the paper of ReActNet, the ReActNet-MobileNet can achieve 69.4% Top-1 Acc on ImageNet datasets, why it does not perform well on CIFAR100?

![image](https://user-images.githubusercontent.com/24189567/142231355-fb90928b-9ced-4894-bb43-efdba730d282.png)

**Comment by szq0214** [comment]
Hi @NicoNico6, thanks very much for your interest in our work! It seems the vanilla MobileNet itself performs not well on CIFAR-100.
https://github.com/NoUnique/MobileNet-CIFAR100.pytorch
https://github.com/weiaicunzai/pytorch-cifar100
This may be caused by its structural property but I think it definitely can be further studied in depth.

**Comment by Xingrun-Xing** [comment]
> Hi @NicoNico6, thanks very much for your interest in our work! It seems the vanilla MobileNet itself performs not well on CIFAR-100. https://github.com/NoUnique/MobileNet-CIFAR100.pytorch https://github.com/weiaicunzai/pytorch-cifar100 This may be caused by its structural property but I think it definitely can be further studied in depth.

你好，cifar10和100你们用多大分辨率训的，另外二值网络from scratch是指reactnet one step的训练结果吗？多谢

**Comment by szq0214** [comment]
> > Hi @NicoNico6, thanks very much for your interest in our work! It seems the vanilla MobileNet itself performs not well on CIFAR-100. https://github.com/NoUnique/MobileNet-CIFAR100.pytorch https://github.com/weiaicunzai/pytorch-cifar100 This may be caused by its structural property but I think it definitely can be further studied in depth.
> 
> 你好，cifar10和100你们用多大分辨率训的，另外二值网络from scratch是指reactnet one step的训练结果吗？多谢

Hi @Xingrun-Xing 分辨率是224x224的因为要match ImageNet上的模型  (finetuning和linear evaluation)，from scratch指one step training。
